packages/ui/src/components/sign-up/sign-up.tsx

```diff
@@ -1,17 +1,22 @@
 import { Root as SignUpRoot } from '@clerk/elements/sign-up';
+import type { SignUpProps } from '@clerk/types';
 
 import { SignUpContinue } from '~/components/sign-up/steps/continue';
 import { SignUpStart } from '~/components/sign-up/steps/start';
 // import { SignUpStatus } from '~/components/sign-up/steps/status';
 import { SignUpVerifications } from '~/components/sign-up/steps/verifications';
+import type { Appearance } from '~/contexts';
+import { AppearanceProvider } from '~/contexts';
```

Comment:
[minor] I think these are equivalent and don't require 2 lines

```suggestion
import { type Appearance, AppearanceProvider } from '~/contexts';
```